src/webapp/src/components/Navbar/index.tsx

```diff
@@ -1,23 +1,39 @@
 import React from 'react';
 import { Button } from '@small-ads/ui';
 import { useHistory, useRouteMatch } from 'react-router-dom';
-import { HeaderStyles, Header } from './navbar.style';
+import { useDispatch, useSelector } from 'react-redux';
+import { signOutUser } from '../../auth/asyncAuthActions';
+import { Header, HeaderStyles } from './navbar.style';
+import { selectUser } from '../../auth/selectors';
 
 export const Navbar = () => {
   const history = useHistory();
   const route = useRouteMatch('/login');
+  const dispatch = useDispatch();
+  const { authStatus } = useSelector(selectUser);
+
+  const handleLogout = () => {
+    dispatch(signOutUser());
+  };
   return (
     <HeaderStyles>
       <Header>
-        <Button
-          disabled={Boolean(route?.isExact)}
-          size='sm'
-          onClick={() => {
-            history.push('/login');
-          }}
-        >
-          Login
-        </Button>
+        {!authStatus && (
+          <Button
+            disabled={Boolean(route?.isExact)}
+            size='sm'
+            onClick={() => {
+              history.push('/login');
+            }}
+          >
+            Login
+          </Button>
+        )}
+        {Boolean(authStatus) && (
```

Comment:
A bit confused here. `authStatus` can be 
```
{
  notLoggedIn,
  loggedIn,
  loading,
}
``` 
How is that in the case `notLoggedIn` we want to show the button? I undestant that this might be impossible now because you are redirecting but still this is a bit cryptic for me here. Can we be explicit in this statement?